apps/voting/app/src/components/AutoLink.js

```diff
@@ -3,9 +3,7 @@ import Linkify from 'react-linkify'
 import { Link } from '@aragon/ui'
 
 const AutoLink = ({ children }) => (
-  <Linkify component={Link} properties={{ target: '_blank' }}>
-    {children}
-  </Linkify>
+  <Linkify component={Link}>{children}</Linkify>
```

Comment:
Doesn't this remove the `target` prop on the link?

Reply:
`Link` applies it by default if an `href` is present